src/audio/component.c

```diff
@@ -74,7 +74,7 @@ struct comp_dev *comp_new(struct sof_ipc_comp *comp)
 		       comp, sizeof(*comp));
 	assert(!ret);
 
-	cdev->drv = drv;
+	cdev->drv = drv; /*this is another assignment, left just in case */
 	list_init(&cdev->bsource_list);
```

Comment:
This comment is confusing here. Since cdev->drv is already set, I think it is better to remove this assignment.

Reply:
Done